transformers/events/log_item_update/transformer.go

```diff
@@ -0,0 +1,64 @@
+package log_item_update
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+	"github.com/makerdao/vulcanizedb/pkg/eth"
+)
+
+const OfferId event.ColumnName = "offer_id"
+
+type Transformer struct{}
+
+func (Transformer) toEntities(contractAbi string, logs []core.EventLog) ([]LogItemUpdateEntity, error) {
+	var entities []LogItemUpdateEntity
+	for _, log := range logs {
+		var entity LogItemUpdateEntity
+		address := log.Log.Address
+		abi, parseErr := eth.ParseAbi(contractAbi)
+		if parseErr != nil {
+			return nil, parseErr
+		}
+
+		contract := bind.NewBoundContract(address, abi, nil, nil, nil)
+		unpackErr := contract.UnpackLog(&entity, "LogItemUpdate", log.Log)
+		if unpackErr != nil {
+			return nil, unpackErr
+		}
+
+		entity.HeaderID = log.HeaderID
+		entity.LogID = log.ID
+		entities = append(entities, entity)
+	}
+
+	return entities, nil
+}
+func (t Transformer) ToModels(abi string, logs []core.EventLog, db *postgres.DB) ([]event.InsertionModel, error) {
```

Comment:
maybe worth replacing `db` with `_`

Reply:
with adding the address column, looks like we'll need the db in the transformer now. but good call!